docs/_docs/deployment/automated.md

```diff
@@ -18,6 +18,7 @@ We have guides for the following providers:
 * [Travis CI]({{ '/docs/continuous-integration/travis-ci/' | relative_url }})
 * [CircleCI]({{ '/docs/continuous-integration/circleci/' | relative_url }})
 * [Buddy]({{ '/docs/continuous-integration/buddyworks/' | relative_url }})
+* [GitHub Actions]({{ '/docs/continuous-integration/github-actions/' | relative_url }})
```

Comment:
```suggestion
* [GitHub Actions]({{ '/docs/continuous-integration/github-actions/' | relative_url }})
* [Travis CI]({{ '/docs/continuous-integration/travis-ci/' | relative_url }})
* [CircleCI]({{ '/docs/continuous-integration/circleci/' | relative_url }})
* [Buddy]({{ '/docs/continuous-integration/buddyworks/' | relative_url }})
```

Let's put it first 😄

Reply:
I didn't want to presume, but yes I like that idea :)